src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx

```diff
@@ -0,0 +1,204 @@
+
+## Get started [#get-started]
+
+Applications don't exist in isolation. The resources you use to monitor, troubleshoot, and improve their performance are both time-critical and complex. You need a simple yet holistic solution to quickly see and act on all your telemetry in context.
+
+Ready to get started?
+
+1. If you don't have one already, [create a New Relic account](https://newrelic.com/signup). It's free, forever.
+2. Update to the [supported agent version](#agents) for your apps and hosts.
+3. [Enable](#agents) telemetry in context for your agent, and make a few small updates to your configuration file.
+
+That's it! Now, to start troubleshooting with relevant telemetry data in your logs, go to New Relic One:
+
+* Explorer UI at [one.newrelic.com](https://one.newrelic.com)
+* Explorer UI for EU region data center if applicable: [one.eu.newrelic.com](https://one.eu.newrelic.com)
+
+
+Your logs will automatically include attributes such as `span.id`, `trace.id`, `hostname`, `entity.guid`, and more. These make it easier to query and facet on related telemetry data.
+
+You can also add more attributes to make your troubleshooting focus even more precise. For example, you may want to add more attributes to get other key telemetry data specific to your ecosystem.
+
+## APM agent configuration [#agents]
+
+After you enable telemetry in context with the supported agent version and update your agent configuration, no additional installation or configuration is required! Your logs will flow into New Relic with related telemetry data for your apps and hosts.
+
+For detailed information about how to enable telemetry in context and update your configuration, see the documentation for your APM agent:
+
+* [Java telemetry in context procedures](/docs/logs/logs-context/java-configure-logs-context-all) for agent [v7.6.0 or higher](/docs/release-notes/agent-release-notes/java-release-notes) for Log4j2 and Logback
+* [.NET telemetry in context procedures](/docs/logs/logs-context/net-configure-logs-context-all) for agent [v0.0.0.0 or higher](/docs/release-notes/agent-release-notes/net-release-notes) for Log4Net
+* [Ruby telemetry in context procedures](/docs/logs/logs-context/configure-logs-context-ruby) for agent [v8.6.0 or higher](/docs/release-notes/agent-release-notes/ruby-release-notes) (default Logger)
+
+If your APM agent does not support our telemetry in context solution, you can continue to use our [standard logs in context solutions](/docs/logs/logs-context/logs-in-context).
+
+## Ensure data privacy [#data-privacy]
+
+Our log management service automatically masks number patterns that appear to be for items such as credit cards or Social Security numbers. For more information, see our [security documentation](/docs/logs/get-started/new-relics-log-management-security-privacy) for log management.
```

Comment:
US alert! Social Security number is a very American thing, could we use something more generic such as national id  or the likes?